app/src/main/java/com/infomaniak/mail/ui/main/newMessage/NewMessageFragment.kt

```diff
@@ -95,6 +98,11 @@ class NewMessageFragment : Fragment() {
     private lateinit var addressListPopupWindow: ListPopupWindow
     private lateinit var filePicker: FilePicker
 
+    private val aiPromptFragment by lazy {
+        // TODO : Instance is recreated when NewMessageFragment in recreated. Why is that? Fix this
```

Comment:
Ask @sirambd if there's any reason for this comportement

Reply:
This is expected, if the Activity is recreated the NewMessageFragment will be another instance and therefore won't share its "by lazy" values